examples/nidaqmx/analog-input-betterproto.py

```diff
@@ -87,17 +87,25 @@ async def raise_if_error(response):
         )
 
         await raise_if_error(await daq_service.start_task(task=task))
+        
+        response = await daq_service.get_task_attribute_u_int32(
```

Comment:
sigh, it seems like this should be `get_task_attribute_uint32` (with no underscore between `u` and `int32`), but I'm not sure it's worth fixing - what do you think?

Reply:
oh, this is generated by `betterproto`, isn't it? Nevermind then :-)

Reply:
Yeah, unfortunately. Can't really blame them for this one because `UInt32` a pure special-case. (They also `_` between acronyms, which is closer to a bug). We could change the method on our end to `Uint32`. But `UInt32` is pretty ingrained at NI and, I guess for that reason, `Uint32` looks weird to me.